scripts/run_benches_for_runtime.sh

```diff
@@ -40,8 +40,8 @@ pallets=(
 	pallet-message-queue
 	cumulus-pallet-parachain-system
 	pallet_multisig
-	pallet-assets
 	pallet-asset-switch
+	pallet-assets
```

Comment:
Just for curiosity, was this the result of running some fmt on this file?

Reply:
No changes :thinking:

Reply:
Could be. I am using shfmt.

Reply:
I don’t think so. If I remember correctly, I initially added only the Peregrine-specific pallets to the pallet list but later reverted that change.

Reply:
Then this change could probably be reverted as well since it's not really useful.

Reply:
reverted in: [afffda3](https://github.com/KILTprotocol/kilt-node/pull/818/commits/afffda333d5e3253f9a6867124444d7325f922fa)